supervisor/misc/tasks.py

```diff
@@ -174,15 +174,9 @@ async def _watchdog_homeassistant_api(self):
             self._cache[HASS_WATCHDOG_API_FAILURES] = 0
             return
 
-        # Give up after 5 reanimation failures in a row. Supervisor cannot fix this issue.
+        # After 5 reanimation attempts switch to safe mode. If that fails, give up
         reanimate_fails = self._cache.get(HASS_WATCHDOG_REANIMATE_FAILURES, 0)
-        if reanimate_fails >= HASS_WATCHDOG_MAX_REANIMATE_ATTEMPTS:
-            if reanimate_fails == HASS_WATCHDOG_MAX_REANIMATE_ATTEMPTS:
-                _LOGGER.critical(
-                    "Watchdog cannot reanimate Home Assistant Core, failed all %s attempts.",
-                    reanimate_fails,
-                )
-                self._cache[HASS_WATCHDOG_REANIMATE_FAILURES] += 1
+        if reanimate_fails > HASS_WATCHDOG_MAX_REANIMATE_ATTEMPTS:
             return
```

Comment:
It does feel a bit weird to me that this is up here, and the rest of the reanimate handling is below, how about moving it further down?

```suggestion
```

Reply:
Not really tbh. The point here is that if we are over our max retry attempts just exit. Don't check for API fails and do any further logging, we know its down and there's nothing more we can do about it.

Reply:
Ok after talking it over we will do this. It will log that the API failed one last time so the user can see in the logs that it is still down and then it will stop logging, that seems fine.